plutus-core/plutus-core/src/PlutusCore/Builtin/Meaning.hs

```diff
@@ -236,37 +227,47 @@ type family ThrowOnBothEmpty binds args isBuiltin a where
             'Text "A built-in function must take at least one type or term argument" ':$$:
             'Text "‘" ':<>: 'ShowType a ':<>: 'Text "’ is a built-in type" ':<>:
             'Text " so you can embed any of its values as a constant" ':$$:
-            'Text "If you still want a built-in function, introduce a dummy ‘()’ argument"
+            'Text "If you still want a built-in function, add a dummy ‘()’ argument"
             )
     ThrowOnBothEmpty '[] '[] 'False a =
         TypeError (
             'Text "A built-in function must take at least one type or term argument" ':$$:
-            'Text "To fix this error introduce a dummy ‘()’ argument"
+            'Text "To fix this error add a dummy ‘()’ argument"
             )
     ThrowOnBothEmpty _ _ _ _ = ()
 
--- See Note [Automatic derivation of type schemes]
--- | Construct the meaning for a built-in function by automatically deriving its
--- 'TypeScheme', given
+-- | A function turned into a type class with exactly one fully general instance.
+-- We can't package up the constraints of 'makeBuiltinMeaning' (see the instance) into a type or
```

Comment:
I was expecting to therefore see you replace some places that have a big load of constraints with just `MakeBuiltinMeaning`, but I can only see on in the debug module?

Reply:
Yep, I just wanted it for `Debug`. It was constantly getting out of sync with the main thing, I got annoyed, tried to create a type/class synonym, failed, created a function-as-a-class. I can revert that if you believe it's not worth the added complexity, but it'll make me sad and what's the point of being right if your developer cries?